test/functionalTest/cases/3068_non_legacy_notifications/non_legacy_notifications_with_accumulator.test

```diff
@@ -0,0 +1,113 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Simple forwarded query in NGSIv2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 38,186-187,235 IPV4
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Register entities of type T, with ID .*, and attribute A1, for the accumulator
```

Comment:
It would be also great to check in a similar fashion (i.e. with accumulator)

* 0x. Register entities of type T, with ID .*, and no attributes, for the accumulator
* 0x. Register entities of type T, with ID E1, and attribute A1, for the accumulator
* 0x. Register entities of type T, with ID E1, and no attributes, for the accumulator

(Maybe already done... I haven't finished to review the .test, but I don't want to forget this comment :)

Reply:
Another one:

0x. Register entities of type T, with ID E1, and attribute A1+A2, for the accumulator

(To check that we see `attrs=A1,A2` in the accumulator)

Reply:
No not done, those I intended to try with real providers.
This functest I only used in the beginning.
But sure, I can add these new cases, doesn't hurt.

Reply:
This is another case of wanting forwarding to be perfect in this PR.
That is not what this PR is about.
For perfect forwarding we need to rewrite the mechanism.
Not a good idea to waste time trying to convert the current mechanism into something else.
Would take me months.
Rewriting it should take a week.

Reply:
It is not a problem to keep it as it is now if the behaviour in NGSIv1 is the same.

Could you check and confirm that in the equivalent NGSIv1 registration case (client doing `GET /v2/entities?id=E1` but registrtion done in NGSIv1 for ID E1 and attribute A1+A2) the accumulator gets `POST /v1/queryContext` on E1 without any attribute?

Reply:
In addition, it seems that the cases I commented:

* 0x. Register entities of type T, with ID .*, and no attributes, for the accumulator
* 0x. Register entities of type T, with ID E1, and attribute A1, for the accumulator
* 0x. Register entities of type T, with ID E1, and no attributes, for the accumulator

has not been yet added (or maybe I have missed them in the new version of the .test?)

Reply:
The first of the three has been added,
As none of them work right now, no meaning to add any more test cases until the problem is fixed.

TBI

Reply:
Fixed in 19efa15ad